src/Microsoft.Data.SqlClient/tests/FunctionalTests/Microsoft.Data.SqlClient.Tests.csproj

```diff
@@ -10,6 +10,7 @@
     <OutputPath>$(BinFolder)$(Configuration).$(Platform).$(AssemblyName)</OutputPath>
   </PropertyGroup>
   <ItemGroup Condition="!$(ReferenceType.Contains('NetStandard'))">
+    <Compile Include="..\..\netfx\src\Microsoft\Data\SqlClient\GenericConverter.cs" Link="GenericConverter.cs" />
```

Comment:
Please remove this from here, as tests also consume NuGet packages when needed and should not include files from project like that.

Reply:
done -- however please review other conversations about how to test the internal class